bundle/community/bundle.Dockerfile

```diff
@@ -15,7 +15,8 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 # Copy files to locations specified by labels.
-COPY bundle/manifests /manifests/
-COPY bundle/metadata /metadata/
-COPY bundle/tests/scorecard /tests/scorecard/
-LABEL com.redhat.openshift.versions="v4.12-v4.15"
+COPY ./manifests /manifests/
+COPY ./metadata /metadata/
+COPY ./tests/scorecard /tests/scorecard/
+
+LABEL com.redhat.openshift.versions=v4.12
```

Comment:
Why did this label change? Was the old value incorrect?

Reply:
Maximum version seems to be not really needed